spec/support/authentication_helpers.rb

```diff
@@ -48,6 +48,8 @@ def submit_credentials(expected_text = '')
         click_button 'Send Me a Push'
       end
     end
+
+    click_button 'Yes, trust browser'
```

Comment:
i wonder if this will be needed for all of us, or if it will cause tests to fail for some of us.  The convoluted logc of L33-50 was for a similar situation.

Reply:
Sounds good, thx for testing